composer.json

```diff
@@ -5,20 +5,21 @@
     "license": "MIT",
     "minimum-stability": "dev",
     "require": {
-        "php": ">=7.2.5",
```

Comment:
What is the reason for bumping the minimum PHP requirement from 7.2.5 -> 7.3.0 ? This would be a BC break of which I'm not sure we're ready to do.